### PR TITLE
feat(curriculum): add event flyer lab

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

images: https://github.com/freeCodeCamp/cdn/pull/278

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+```js
+assert(new __helpers.CSSHelp(document).getStyle('section')?.width.endsWith('%'));
+```
+
+# --seed--
+
+## --seed-contents--
+
+```html
+
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md:129]:
```suggestion
<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>Build an Event Flyer Page</title>
</head>

<body>

</body>

</html>
```

File: client/i18n/locales/english/intro.json [hunk 2]
```diff
@@ -1799,7 +1799,12 @@
       "hgxz": { "title": "52", "intro": [] },
       "bgrw": { "title": "53", "intro": [] },
       "sbnz": { "title": "54", "intro": [] },
-      "chca": { "title": "55", "intro": [] },
+      "lab-event-flyer-page": {
+        "title": "Build an Event Flyer Page",
+        "intro": [
+          "In this lab, will use absolute and relative CSS units to create an event flyer page."
```
Comment (ilenia-magoni) [inline client/i18n/locales/english/intro.json:1805]:
```suggestion
          "In this lab, you will use absolute and relative CSS units to create an event flyer page."
```

File: client/src/pages/learn/front-end-development/lab-event-flyer-page/index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: Introduction to the Build an Event Flyer Page
+block: lab-event-flyer-page
+superBlock: front-end-development
+---
+
+## Introduction to the Build an Event Flyer Page
+
+In this lab, will use absolute and relative CSS units to create an event flyer page.
```
Comment (ilenia-magoni) [inline client/src/pages/learn/front-end-development/lab-event-flyer-page/index.md:9]:
```suggestion
In this lab, you will use absolute and relative CSS units to create an event flyer page.
```

File: curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+1. You should have a `header` element within the body.
+1. Your `header` element should have an image in it for your event, and an `h1` in it with your event title, in that order. You can use this image if you would like: `https://cdn.freecodecamp.org/curriculum/labs/event.jpg`
+1. You should have a `main` element within the body.
+1. Your `main` element should have at least two `section` elements within it showcasing the event features.
+1. Your body should have a top and bottom padding of `50px`.
+1. Your body should have a top and bottom margin of `0`, and a left and right margin that centers itself.
+1. Your body should have a width set relative to the width of the viewport.
+1. Your body should use the `calc` function to set its height to 100% of the viewport's height minus any padding applied to the top and bottom of the body.
+1. You should have at least one `hr` within your flyer.
+1. You should the `width` of all `hr` and `section` elements to a percent value relative to its parent.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md:24]:
```suggestion
1. You should set the `width` of all `hr` and `section` elements to a percent value relative to its parent.
```

I am not sure it is the right word, but a word is missing

File: curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+
+# --hints--
+
+You should have a `header` element within the `body` element.
+
+```js
+assert.exists(document.querySelector('body > header'));
+```
+
+You should have an `img` element as the first child in the header.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md:34]:
```suggestion
You should have an `img` element as the first child in the `header`.
```

File: curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+
+```js
+assert.equal(document.querySelector('header')?.children?.[0].tagName, 'IMG');
+```
+
+You should have an `h1` element as the second child in the header.
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-event-flyer-page/66e45c8140f9fda5c105ae26.md:40]:
```suggestion
You should have an `h1` element as the second child in the `header`.
```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]

**Review by Dario-DC — COMMENTED** [review]
I was able to pass all the tests with almost no content showing on the page. I think we should add some additional test, at least to verify that the image has a `src` set and the two sections are not completely empty.

**Comment by moT01** [comment]
Thanks for taking a look @ilenia-magoni & @Dario-DC 👍 I added the suggestions.

**Review by Dario-DC — APPROVED** [review]
LGTM :+1:

**Review by jdwilkin4 — APPROVED** [review]

**Review by majestic-owl448 — APPROVED** [review]
